instrumentation/java11-http-client/java-http-client/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    coreJdk()
+  }
+}
+
+dependencies {
+  compileOnly project(":bootstrap")
+  compileOnly project(":utils")
+
+  compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv")
+
+  implementation project(":instrumentation:java11-http-client:java-http-client-config")
+
+  // Not needed as we've directly included the contents of the instrumentation library
+  // implementation("io.opentelemetry.instrumentation:opentelemetry-java-http-client")
```

Comment:
keeping this comment?

Reply:
I wanted to make it clear that we've copied the code from upstream, but embedded the content of that dependency directly to override features